src/plugins/dashboard/server/saved_objects/dashboard_migrations.ts

```diff
@@ -101,17 +101,7 @@ const migrations700: SavedObjectMigrationFn<any, any> = (doc): DashboardDoc700To
 };
 
 export const dashboardSavedObjectTypeMigrations = {
-  /**
-   * We need to have this migration twice, once with a version prior to 7.0.0 once with a version
-   * after it. The reason for that is, that this migration has been introduced once 7.0.0 was already
-   * released. Thus a user who already had 7.0.0 installed already got the 7.0.0 migrations below running,
-   * so we need a version higher than that. But this fix was backported to the 6.7 release, meaning if we
-   * would only have the 7.0.1 migration in here a user on the 6.7 release will migrate their saved objects
-   * to the 7.0.1 state, and thus when updating their Kibana to 7.0, will never run the 7.0.0 migrations introduced
-   * in that version. So we apply this twice, once with 6.7.2 and once with 7.0.1 while the backport to 6.7
-   * only contained the 6.7.2 migration and not the 7.0.1 migration.
-   */
-  '6.7.2': flow(migrateMatchAllQuery),
   '7.0.0': flow(migrations700),
   '7.3.0': flow(migrations730),
+  '7.8.2': flow(migrateMatchAllQuery),
```

Comment:
There is no plan for a 7.8.2 release, so we should make this 7.9.3 instead, since it's the oldest version it'll go in.